README.md

```diff
@@ -124,55 +124,35 @@ Uncomment and modify the below changes in Hiera file to apply Secure Vault.
     wso2::enable_secure_vault: true
     ```
 
-2. Add Secure Vault configurations as below
+2. Run the ciphertool.sh / ciphertool.bat (in <WSO2_HOME>/bin) with -Dconfigure parameter and enter the primary keystore password of carbon server.
 
-    ```yaml
-    wso2::secure_vault_configs:
-      <secure_vault_config_name>:
-        secret_alias: <secret_alias>
-        secret_alias_value: <secret_alias_value>
-        password: <password>
-    ```
-
-    Ex:
-    ```yaml
-    wso2::secure_vault_configs:
-      key_store_password:
-        secret_alias: Carbon.Security.KeyStore.Password
-        secret_alias_value: repository/conf/carbon.xml//Server/Security/KeyStore/Password,false
-        password: wso2carbon
-    ```
-
-    For Identity Server `5.0.0` which is based on WSO2 Carbon Kernel 4.2.0
-
-    Ex:
-    ```yaml
-    wso2::secure_vault_configs:
-      key_store_password:
-        secret_alias: Carbon.Security.KeyStore.Password
-        secret_alias_value: carbon.xml//Server/Security/KeyStore/Password,true
-        password: wso2carbon
-    ```
+```bash
+sh ciphertool.sh -Dconfigure
+[Please Enter Primary KeyStore Password of Carbon Server : ]
```

Comment:
This is a prompt we get when executing `sh ciphertool.sh -Dconfigure` and is related the carbon secure vault